js/main.js

```diff
@@ -381,14 +395,18 @@ $(document).ready(function() {
     L.control.locate({keepCurrentZoomLevel: true}).addTo(map);
 
     // Populate dropdown
-    loadCityIDs().done(function(cityIDs) {
-        for (var i = 0; i < cityIDs.length; i++) {
-            var id = cityIDs[i];
+    loadCities().fail(function() {
+        console.log("Loadcity(); failed.");
```

Comment:
Would be nice if the error would state the actual problem. Can you pass in the error message from the calling side?